setup.py

```diff
@@ -66,7 +66,7 @@
     description="Craft provider tooling",
     long_description=readme,
     author="Canonical Ltd.",
-    author_email="Canonical Ltd.",
+    author_email="chris.patterson@canonical.com",
```

Comment:
I would probably use snapcraft@lists.snapcraft.io which is hooked up to the forum so we all get the emails.

Reply:
done